pkg/helm/helm.go

```diff
@@ -56,28 +56,30 @@ func CreateOrUpdateHelmIndex(rootFs billy.Filesystem) error {
 }
 
 // UpdateIndex updates the original index with the new contents
-func UpdateIndex(original, new *helmRepo.IndexFile) (*helmRepo.IndexFile, bool) {
+// !! modifies newRepo in place and returns it
+func UpdateIndex(origRepo, newRepo *helmRepo.IndexFile) (*helmRepo.IndexFile, bool) {
```

Comment:
this does now modify `newRepo` in place which could cause side effects, but I vetted the current codebase and newRepo struct is never used after a call to `UpdateIndex`